### PR TITLE
Add comment about auto exportRuntime in compiler.md

### PR DESCRIPTION
**Pull request by MaxGraey**
See https://github.com/AssemblyScript/assemblyscript/pull/2383

### CODE REVIEW

**Comment by dcodeIO** [comment]
Looks like the description should first be added to the CLI options, and then copied to documentation?

**Comment by MaxGraey** [comment]
Yeah. Good point!

File: src/compiler.md [hunk 1]
```diff
@@ -96,8 +96,8 @@ There are several flags that enable or disable specific WebAssembly or compiler
                         stub         Minimal runtime stub (never frees)
                         ...          Path to a custom runtime implementation
 
---exportRuntime       Always exports the runtime helpers (__new, __collect etc.).
-                      When host `bindings` is enabled it will be determined automatically.
+--exportRuntime       Always exports the runtime helpers (__new, __collect, __pin etc.).
+                      Automatically determined when generation of --bindings is enabled.
```
Comment (dcodeIO) [inline src/compiler.md:100]: Only noticing now: I think the lines here have become (too) long?

Comment (MaxGraey) [reply src/compiler.md:100]: No, I guess it's ok

**Pull request merged**